src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const baseRateOneDay = 40;
+  const THREE_DAYS = 3;
+  const SEVEN_DAYS = 7;
+  const DISCOUNT_20 = 20;
+  const DISCOUNT_50 = 50;
+  let totalCost = days * baseRateOneDay;
+
+  if (days >= THREE_DAYS && days < SEVEN_DAYS) {
+    totalCost -= DISCOUNT_20;
+  } else if (days >= SEVEN_DAYS) {
+    totalCost -= DISCOUNT_50;
```

Comment:
There is no need for the `else` keyword here as you are using `return` inside the `if` block. This can simplify the control flow and make the code easier to read.